### ISSUE TITLE
Horizontal scroll bug

### ISSUE THREAD

**Issue by kaermorchen** [Simonwep/pickr#50]
#### Do you want to request a **feature** or report a **bug**? 
A bug.

#### What is the current behavior?
If the color input has positions about right side. Browser displays horizontal scroll after initial pickr. If I open and close pickr the horizontal scroll leaves. Looks like the problem is `pcr-app` don't has `display: none;` property

![2018-11-17_15-13-21 2](https://user-images.githubusercontent.com/11972062/48660967-a41c5f80-ea7b-11e8-8df0-a8eb69f67ac3.png)

#### If the current behavior is a bug, please provide the steps to reproduce.
Move input to the right side.

#### What is the expected behavior?
Without the horizontal scroll.

#### Your environment:
```
Pickr-version: 0.3.1
Browser-version:  Firefox 63.0.3
Operating-system:  Windows 7 64
```

**Comment by Simonwep** [comment]
I've reproduced it and (hopefully) fixed it here 2c7aeef396ba9194165b9fd0f4f5c09e517ada14 :)

**Comment by kaermorchen** [comment]
I confirm, the bug is fixed. Thank you. Your lib is awesome.
